pennylane/operation.py

```diff
@@ -464,7 +464,6 @@ def __init__(self, *params, wires=None, do_queue=True, id=None):
         self._name = self.__class__.__name__  #: str: name of the operator
         self._id = id
         self.queue_idx = None  #: int, None: index of the Operator in the circuit queue, or None if not in a queue
-        self._hyperparameters = {}
```

Comment:
Now that `_hyperparameters` is created the first time `hyperparameters` is called, we don't need to initialize here.

Reply:
True, this was bugging me too!